includes/services.inc.php

```diff
@@ -129,7 +129,7 @@ function poll_service($service)
 
     // If we do not have a cmd from the check script, build one.
     if ($check_cmd == "") {
-        $check_cmd = $config['nagios_plugins'] . "/check_" . $service['service_type'] . " -H " . ($service['service_ip'] ? $service['service_ip'] : $service['hostname']);
```

Comment:
You're using them wrong.

If the device you want to do the check against is the hostname you've added the device as then don't set any parameters or the IP. If you want to use a different hostname or IP then set it in the IP box. You don't need to use -H in the parameters box at all. Removing the code you have done means you will break existing service checks for most people.

If a service check doesn't need the -H then you create a custom service check file like we already have, see `includes/services/check_procs.inc.php`